hazelcast/src/test/java/com/hazelcast/config/XMLConfigPreProcessorTest.java

```diff
@@ -213,10 +186,7 @@ public void testImportEmptyResourceThrowsException() throws Exception {
         String xml = "<hazelcast>\n" +
                 "    <import resource=\"\"/>\n" +
                 "</hazelcast>";
-
-        ByteArrayInputStream bis = new ByteArrayInputStream(xml.getBytes());
-        XmlConfigBuilder configBuilder = new XmlConfigBuilder(bis);
-        configBuilder.build();
+        buildConfig(xml, null);
```

Comment:
I'm not going to be very problematic about this; but normally I would create an additional buildConfig method which doesn't need a null. And perhaps it just forwards to buildConfig(xml, null)... but at least you don't see it in the code.

You don't need to change it, but passing in null values to an argument always gives me an strange feeling + decreases readability of the test because the reader is going to think.. what is going on here! And he is getting confused..